File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/examples-java/ApiManagementPortalConfig.java
/** Samples for PortalConfig Get. */
public final class Main {
    /*
     * x-ms-original-file: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/examples/ApiManagementPortalConfig.json
     */
    /**
     * Sample code: ApiManagementPortalConfig.
     *
     * @param manager Entry point to ApiManagementManager.
     */
    public static void apiManagementPortalConfig(com.azure.resourcemanager.apimanagement.ApiManagementManager manager) {
        manager.portalConfigs().getWithResponse("rg1", "apimService1", "default", com.azure.core.util.Context.NONE);
    }
}

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/examples-js/ApiManagementPortalConfig.js
const { ApiManagementClient } = require("@azure/arm-apimanagement");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Get the developer portal configuration.
 *
 * @summary Get the developer portal configuration.
 * x-ms-original-file: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/examples/ApiManagementPortalConfig.json
 */
async function apiManagementPortalConfig() {
  const subscriptionId = process.env["APIMANAGEMENT_SUBSCRIPTION_ID"] || "subid";
  const resourceGroupName = process.env["APIMANAGEMENT_RESOURCE_GROUP"] || "rg1";
  const serviceName = "apimService1";
  const portalConfigId = "default";
  const credential = new DefaultAzureCredential();
  const client = new ApiManagementClient(credential, subscriptionId);
  const result = await client.portalConfig.get(resourceGroupName, serviceName, portalConfigId);
  console.log(result);
}
